src/utils/potentialDupMsg/index.ts

```diff
@@ -0,0 +1,34 @@
+import { Message } from '@glif/filecoin-message'
+import BigNumber from 'bignumber.js'
+import { Message as GQLMessage } from '../../generated/graphql'
+import { isAddrEqual } from '../isAddrEqual'
+
+export type ComparisonMessage = Pick<
+  GQLMessage,
+  'cid' | 'from' | 'to' | 'value' | 'params' | 'method'
+> & { height: string | number }
+
+export const potentialDupMsg = (
+  msg1: ComparisonMessage,
+  msg2: Message,
+  currentHeight: number,
+  isPending: boolean = false
+): string | null => {
+  if (!msg1 || !msg2) return null
+
+  const fromAddrEq =
+    isAddrEqual(msg1.from, msg2.from) || isAddrEqual(msg1.from, msg2.from)
+  const toAddrEq =
+    isAddrEqual(msg1.to, msg2.to) || isAddrEqual(msg1.to, msg2.to)
```

Comment:
Doesn't one isAddrEqual check for from and to suffice? They seem identical

Reply:
Oh yeah good catch!